### PR TITLE
Added validate user tags, Jenkins nightly builds

### PR DESCRIPTION
**Pull request by athiruma**
1. Created the validated IAM user tags and trailing spaces.
2. Added cost-explorer Jenkins Nightly build for PSAP, perf-scale accounts.
3. Added validated tags, and trailing spaces Jenkins nightly builds.

### CODE REVIEW

File: cloud_governance/tag_user/iam_user_tags.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    def upload_trailing_user_tags(self):
+        """
+        This method gets the trailing spaces tags of each user
+        @return:
+        """
+        users = self.iam_operations.get_users()
+        output_data = []
+        for user in users:
+            if '-' not in user['UserName']:
```
Comment (ebattat) [inline cloud_governance/tag_user/iam_user_tags.py:40]: For what we need this check ?

File: jenkins/run_upload_es.py [hunk 2]
```diff
@@ -5,13 +5,17 @@
...
+AWS_SECRET_ACCESS_KEY_DELETE_PSAP = os.environ['AWS_SECRET_ACCESS_KEY_DELETE_PSAP']
 BUCKET_PERF = os.environ['BUCKET_PERF']
 AWS_ACCESS_KEY_ID_PSAP = os.environ['AWS_ACCESS_KEY_ID_PSAP']
 AWS_SECRET_ACCESS_KEY_PSAP = os.environ['AWS_SECRET_ACCESS_KEY_PSAP']
 BUCKET_PSAP = os.environ['BUCKET_PSAP']
 AWS_ACCESS_KEY_ID_RH_PERF = os.environ['AWS_ACCESS_KEY_ID_RH_PERF']
 AWS_SECRET_ACCESS_KEY_RH_PERF = os.environ['AWS_SECRET_ACCESS_KEY_RH_PERF']
 BUCKET_RH_PERF = os.environ['BUCKET_RH_PERF']
+AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE = os.environ['AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE']
+AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE = os.environ['AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE']
```
Comment (ebattat) [inline jenkins/run_upload_es.py:18]: Is it enable also non cluster delete ? (volumes)

File: cloud_governance/tag_user/iam_user_tags.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/tag_user/iam_user_tags.py:40]: We know User have without **-.**

File: jenkins/run_upload_es.py (hunk 2 above)
Comment (athiruma) [reply jenkins/run_upload_es.py:18]: No.

File: cloud_governance/tag_user/iam_user_tags.py (hunk 1 above)
Comment (ebattat) [reply cloud_governance/tag_user/iam_user_tags.py:40]: You mean that the user value doesnt include -. right 
What will be if we have a user with this char ?

Comment (athiruma) [reply cloud_governance/tag_user/iam_user_tags.py:40]: We create a user without -, With - means it's not created by us manually. except for our cloud-governance

Comment (ebattat) [reply cloud_governance/tag_user/iam_user_tags.py:40]: DO we not have other way to find it ?

Comment (athiruma) [reply cloud_governance/tag_user/iam_user_tags.py:40]: we do it same for IAM user tags update.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**